test-data/unit/check-newtype.test

```diff
@@ -337,7 +339,7 @@ class C(B): pass  # E: Cannot subclass NewType
 from typing import Protocol, NewType
 
 class P(Protocol):
-    attr: int
+    attr: int = 0
```

Comment:
What's this change?

Reply:
As I explained offline, there was previously a false negative in this test (the NewType is abstract otherwise).